core/os/os.cpp

```diff
@@ -441,12 +441,15 @@ bool OS::has_feature(const String &p_feature) {
 	}
 	if (p_feature == "editor_hint") {
 		return _in_editor;
-	} else if (p_feature == "editor_runtime") {
+	} else if (p_feature == "editor_runtime" || p_feature == "runtime") {
```

Comment:
i think this comparison is breaking this usecase:
```
@if_feature("editor_runtime")
func _process(delta: float) -> void:
>   # editor only process
@if_feature("runtime")
func _process(delta: float) -> void:
>   # runtime only process
```

Reply:
`editor_runtime` means the game being run from a tools-enabled build.

For editor-only process, you'd use `editor_hint`.

Reply:
i see